src/providers.jl

```diff
@@ -48,6 +73,11 @@ end
 # If we're using fftw_jll, load it in
 @static if fftw_provider == "fftw"
     import FFTW_jll
+    if !FFTW_jll.is_available()
+        # more descriptive error message if FFTW is not available
+        # (should not be possible to reach this)
+        error("FFTW library cannot be loaded: please switch to the `mkl` provider for FFTW.jl")
```

Comment:
Maybe provide a more helpful error message to explain how to to do this?  Most users won't have a clue of what this is about.